pkgs/applications/misc/keepassx/community.nix

```
@@ -67,6 +69,7 @@ stdenv.mkDerivation rec {
   doCheck = true;
   checkPhase = ''
     export LC_ALL="en_US.UTF-8"
+	export QT_PLUGIN_PATH="${qtbase.bin}/${qtbase.qtPluginPrefix}"
```

```suggestion
    export QT_PLUGIN_PATH="${qtbase.bin}/${qtbase.qtPluginPrefix}"
    export QT_QPA_PLATFORM=offscreen
```

Convert tab to spaces.

The check phase failed for me. It passed when I used `QT_QPA_PLATFORM=offscreen`.